spacy_llm/tasks/lemma/task.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Callable, Dict, Generic, Iterable, List, Optional, Type
+
+import jinja2
+from spacy.language import Language
+from spacy.scorer import Scorer
+from spacy.tokens import Doc
+from spacy.training import Example
+
+from ...ty import FewshotExample, TaskResponseParser
+from ..util import ExampleType, SerializableTask
+from .util import DEFAULT_LEMMA_TEMPLATE_V1
+
+
+class LemmaTask(SerializableTask[ExampleType], Generic[ExampleType]):
+    def __init__(
+        self,
+        template: str = DEFAULT_LEMMA_TEMPLATE_V1,
+        parse_responses: Optional[TaskResponseParser] = None,
+        fewshot_example_type: Optional[Type[FewshotExample]] = None,
```

Comment:
Again, not sure how this can be `Optional` if the code asserts it to be not `None`?

Reply:
It's set in `__init__()`.

Reply:
Hm, ok. Should we move the `assert` to `__init__()` though?

Reply:
Done in https://github.com/explosion/spacy-llm/pull/242/commits/0f869a729b46ffbfb9e23fcf57b63be1ce1adbd4.